cmd/install/install_render.go

```diff
@@ -25,23 +25,31 @@ const (
 var (
 	RenderFormatYaml = "yaml"
 	RenderFormatJson = "json"
-
-	TemplateParamHyperShiftImage          = "OPERATOR_IMG"
-	TemplateParamHyperShiftImageTag       = "IMAGE_TAG"
-	TemplateParamNamespace                = "NAMESPACE"
-	TemplateParamOIDCS3Name               = "OIDC_S3_NAME"
-	TemplateParamOIDCS3Region             = "OIDC_S3_REGION"
-	TemplateParamOIDCS3CredsSecret        = "OIDC_S3_CREDS_SECRET"
-	TemplateParamOIDCS3CredsSecretKey     = "OIDC_S3_CREDS_SECRET_KEY"
-	TemplateParamAWSPrivateRegion         = "AWS_PRIVATE_REGION"
-	TemplateParamAWSPrivateCredsSecret    = "AWS_PRIVATE_CREDS_SECRET"
-	TemplateParamAWSPrivateCredsSecretKey = "AWS_PRIVATE_CREDS_SECRET_KEY"
-	TemplateParamOperatorReplicas         = "OPERATOR_REPLICAS"
-	TemplateParamExternalDNSCredsSecret   = "EXTERNAL_DNS_CREDS_SECRET"
-	TemplateParamExternalDNSDomainFilter  = "EXTERNAL_DNS_DOMAIN_FILTER"
-	TemplateParamExternalDNSTxtOwnerID    = "EXTERNAL_DNS_TXT_OWNER_ID"
 )
 
+var openshiftTemplateParams = TemplateParams{
+	HyperShiftImage:             "OPERATOR_IMG",
+	Namespace:                   "NAMESPACE",
+	HypershiftOperatorReplicas:  "OPERATOR_REPLICAS",
+	OIDCS3Name:                  "OIDC_S3_NAME",
+	OIDCS3Region:                "OIDC_S3_REGION",
+	OIDCS3CredsSecret:           "OIDC_S3_CREDS_SECRET",
+	OIDCS3CredsSecretKey:        "OIDC_S3_CREDS_SECRET_KEY",
+	AWSPrivateRegion:            "AWS_PRIVATE_REGION",
+	AWSPrivateCredsSecret:       "AWS_PRIVATE_CREDS_SECRET",
+	AWSPrivateCredsSecretKey:    "AWS_PRIVATE_CREDS_SECRET_KEY",
+	ExternalDNSCredsSecret:      "EXTERNAL_DNS_CREDS_SECRET",
+	ExternalDNSDomainFilter:     "EXTERNAL_DNS_DOMAIN_FILTER",
+	ExternalDNSTxtOwnerID:       "EXTERNAL_DNS_TXT_OWNER_ID",
+	ExternalDNSImage:            "EXTERNAL_DNS_IMAGE",
+	RegistryOverrides:           "REGISTRY_OVERRIDES",
+	AROHCPKeyVaultUsersClientID: "AZURE_KEYVAULT_CLIENT_ID",
```

Comment:
are these changes related to helm or is this using the PR to update this command with newest fields? If so can we please split commits accordingly?

Reply:
ack. working on it

Reply:
i've split the work into two commits.
1) introduce helm generator while not changing anything on the output of the openshift template generator
2) introduce new fields